dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -405,8 +407,27 @@ object LagomPlugin extends AutoPlugin {
   }
 
   private def runAllMicroservicesTask: Initialize[Task[Unit]] = Def.taskDyn {
+
     val projects = microservicesProjects.value
-    val filter = ScopeFilter(inProjects(projects: _*))
+
+    // check if application.conf is present in service
+    def isValidProject(p: ProjectRef) = {
+
+      val prj = Project.getProject(p, buildStructure.value)
+
+      val r: PathFinder = {
+        (new File(prj.get.base.getPath) / "src") ** "application.conf"
```

Comment:
I'm thinking it may be better to run the validation step before starting services. After all, if a user types `runAll`, he expects all services to be started. If one or more can't be started, I guess he will want to 1) stop all running services, and 2) fix the problem before re-executing `runAll`. Hence, let's save users from doing step 1, and simply check that all Lagom projects have an application.conf in their classpath before attempting to start them.

I'd also like to hear @jroper opinion.